packages/web/src/pages/upload-page/UploadPageNew.tsx

```diff
@@ -55,7 +70,14 @@ export const UploadPageNew = (props: UploadPageProps) => {
         <SelectPageNew
           tracks={tracks}
           setTracks={setTracks}
-          onContinue={() => setPhase(Phase.EDIT)}
+          onContinue={() => {
+            setPageTitle(
```

Comment:
shouldn't the phase + tracks be enough to determine the pageTitle? ie we dont need a separate state for it?